pvlib/floating.py

```diff
@@ -0,0 +1,70 @@
+"""
+The ``floating`` module contains functions for calculating parameters related
+to floating PV systems.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def stream_temperature_stefan(temp_air):
+    r"""
+    Estimate daily stream water temperature from daily ambient air temperature.
+
+    Parameters
+    ----------
+    temp_air : numeric
+        Daily average ambient dry bulb temperature. [°C]
+
+    Returns
+    -------
+    water_temperature : numeric
+        Daily average stream water temperature. [°C]
+
+    Notes
+    -----
+    Daily average stream water temperature
+    :math:`T_w` is calculated from daily ambient air temperature
+    :math:`T_{air}` as provided in [1]_:
+
+    .. math::
+       :label: stream
+
+        T_w = 5 + 0.75 \cdot T_{air}
+
+    The predicted daily stream water temperatures had a
+    standard deviation of 2.7 °C compared to measurements. Small, shallow
+    streams had smaller deviations than large, deep rivers.
+
+    It should be noted that this equation is limited to streams, i.e., water
+    bodies that are well mixed in the vertical and transverse directions of a
+    cross section. Also, it is only tested on ice-free streams. Consequently,
+    when the mean ambient air temperature is lower than -6.66 °C, the surface
+    stream water temperature is returned as NaN.
+
+    While this equation was created for estimating stream temperature,
+    there are a number of publications that have used it for estimating the
+    lakewater and seawater temperature for floating PV, although without
+    providing any proof of the formula's validity for such purposes.
```

Comment:
Nothing really wrong with the original, but I was just wondering whether we could make this sentence more concise. This is not a 'strong' suggestion though, feel free to ignore it. Please keep your version if you prefer.
```suggestion
    Although this model was developed to estimate stream temperature,
    a number of publications have used it to estimate lakewater and
    seawater temperatures for floating PV without evidence of validity
    for such applications.
```